ruby/vampires.rb

```diff
@@ -0,0 +1,102 @@
+#Coding program to gather informmation from user
+puts "what is the total number of applicants today"
+TOTALAPPLICANTS = gets.chomp.to_i
+
+
+
+
+#Conditional Logic
+index = 0
+
+while index < TOTALAPPLICANTS 
+
+	puts "What is your name?"
+	e_name = gets.chomp.downcase
+
+	puts "How old are you?"
+	e_age = gets.chomp.to_i
+
+	puts "What year were you born"
+	e_birthyear = gets.chomp.to_i
+
+	puts "Would you like us to order you some complimentary garlic bread from our cafeteria?"
+	e_bread = gets.chomp
+
+	puts "Would you like to enroll in the company's health insurance?"
+	e_insurance = gets.chomp
+
+	puts "What are you allergies. Type done when finished entering allergies"
+	e_allergy = gets.chomp.downcase
+
+		until ((e_allergy == "sunshine") || (e_allergy == "done"))
+			e_allergy = gets.chomp.downcase
+		end
+
+
+			if (e_allergy == "sunshine")
+				puts "Probably a vampire"
+			elsif (e_age == (2016 - e_birthyear)) && ((e_bread == "yes") && (e_insurance == "yes") && (e_name == "drake cula") || (e_name == "tu fang"))

```

Comment:
In this case if the name is Fang or Drake then we know this person is a vampire so logically this would be the first thing you should consider checking. Is there any reason to check anymore of the code if we know they are a vampire already?